spec/codecs/nmap_spec.rb

```diff
@@ -27,6 +27,15 @@
           expect(event).to be_a(LogStash::Event)
         end
       end
+
+      let(:ids) { subject.map {|e| e["id"] } }
+      it "should add a unique id field to all events" do
+        expect(ids).to eql(ids.uniq)
+      end
+
+      it "should not have any null id fields" do
+        expect(ids.include?(nil)).to be_falsey
```

Comment:
We're just asserting that all documents should have IDs, in a previous commit I didn't have them, and it was bad. This will be good to have in case someone removes that by accident or adds a new type.